boards/BMS/Src/App/App_Soc.c

```diff
@@ -0,0 +1,48 @@
+#include <math.h>
+#include "App_Soc.h"
+
+ExitCode App_Soc_Vote(
+    float  max_difference,
+    float  soc_1,
+    float  soc_2,
+    float  soc_3,
+    float *result)
+{
+    if (max_difference < 0.0f || max_difference > 100.0f)
+        return EXIT_CODE_INVALID_ARGS;
+
+    if (soc_1 < 0.0f || soc_1 > 100.0f)
+        return EXIT_CODE_INVALID_ARGS;
+
+    if (soc_2 < 0.0f || soc_2 > 100.0f)
+        return EXIT_CODE_INVALID_ARGS;
+
+    if (soc_3 < 0.0f || soc_3 > 100.0f)
+        return EXIT_CODE_INVALID_ARGS;
```

Comment:
In general, I think the `if` without brackets is a tad dangerous because it's easy to accidentally add lines and not realize they're not under the `if`. Could we add brackets just to be super safe?

Reply:
As another thought, if we're concerned that takes up too many lines, we could consider different formattings for it via `clang-format`.

Reply:
No problem, added the brackets